rwf/src/controller/auth.rs

```diff
@@ -336,7 +336,7 @@ impl Authentication for SessionAuth {
         if let Some(ref redirect) = self.redirect {
             Ok(Response::new().redirect(redirect))
         } else {
-            Ok(Response::forbidden())
+            Ok(Response::unauthorized("Basic"))
```

Comment:
`Basic` would imply the user can auth with HTTP Basic Auth, which is not the case. Maybe we can specify `Session` here instead?

Docs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate

Reply:
I don't think `Session` is a supported auth method: https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml. I can change it to leave `WWW-Authenticate` header in these cases if that's preferable.

Reply:
I think that makes sense. I'm not too familiar with what the HTTP spec requires in this case.